DEVELOPMENT_ENVIRONMENT.md

```diff
@@ -75,8 +75,25 @@
 	- under "Eclipse Formatter config file", select ``sormas-base/java-formatter-profile.xml``
 	- check optimize imports and, for "Import order", select ``sormas-base/java-importorder-profile.importorder``
 	- **Important:** select "Do not format other file types by IntelliJ formatter"
-	- go to Preferences -> Editor -> Code style -> Java -> Imports: set class and static names counts for import with * to 99
+	- go to Settings -> Editor -> Code style -> Java -> Imports: set class and static names counts for import with * to 99
 	- for IntelliJ, code formatting is usually done with Ctrl+Alt+L. For automatic formatting, it's recommended to use the plugin Save Actions and check the first three checkboxes in "General" and the first two checkboxes in "Formatting Actions" (https://plugins.jetbrains.com/plugin/7642-save-actions)
+### Enable debugging of the application from IntelliJ side
+**Note: This is only needed for development of the SORMAS back and front apps in order to enable debugging
+-> Download Intellij Idea newer that 15 of April 2020. Version prior to 15 of April is containing a bug which will not allow to enable debugging
```

Comment:
Add this remark to the first line of the IntelliJ paragraph:
Install the latest Ultimate edition IntelliJ. (newer than version of 2020-04-15 to enable debugging, see https://youtrack.jetbrains.com/issue/IDEA-216528).

Remove the subparagraph